### PR TITLE
Improve stack documentation

### PR DESCRIPTION
**Pull request by gaelL**

### CODE REVIEW

File: terraform/module-magento/variables.tf [hunk 1]
```diff
@@ -156,7 +153,7 @@ variable "elasticache_engine" {
 }
 
 variable "elasticache_parameter_group_name" {
-  default = "default.redis3.2"
+  default = "default.redis4.0"
```
Comment (sdurrheimer) [inline terraform/module-magento/variables.tf]: Magento doesn't support `Redis 4.0` officially https://devdocs.magento.com/guides/v2.3/cloud/project/project-conf-files_services-redis.html

Comment (gaelL) [reply terraform/module-magento/variables.tf]: nice catch, I fixed it back to 3.2

File: README.md [hunk 2]
```diff
@@ -4,53 +4,136 @@ Service catalog magento stack
 
 This stack will deploy a Magento on X Amazon EC2 instances behind an ELB load balancer, using RDS database and ElasticCache. 
 
+
+# Architecture
+
 <img src="https://raw.githubusercontent.com/cycloid-community-catalog/stack-magento/master/diagram.jpeg" width="400">
```
Comment (sdurrheimer) [inline README.md]: Could we try to add `align="middle"` ?

```suggestion
<img src="https://raw.githubusercontent.com/cycloid-community-catalog/stack-magento/master/diagram.jpeg" width="400" align="middle">
```

File: README.md [hunk 3]
```diff
@@ -4,53 +4,136 @@ Service catalog magento stack
...
-Terraform job similar to the plan one, but will actually create/update everything that needs to. Please see the plan diff for a better understanding.
+> **Note** The pipeline contains a manual approval between terraform plan and terraform apply.
+> That means if you trigger a terraform plan, to apply it, you have to go on terraform apply job
+> and click on the `+` button to trigger it.
 
-## /!\ Destroy /!\
-**terraform-destroy:**
-Terraform job meant to destroy the whole stack - **NO CONFIRMATION ASKED**. If triggered, the full project **WILL** be destroyed.
-Use with caution.
+<img src="https://raw.githubusercontent.com/cycloid-community-catalog/stack-magento/gl_docs/pipeline.png" width="800">
```
Comment (sdurrheimer) [inline README.md]:
```suggestion
<img src="https://raw.githubusercontent.com/cycloid-community-catalog/stack-magento/gl_docs/pipeline.png" width="800" align="middle">
```

File: README.md [hunk 4]
```diff
@@ -4,53 +4,139 @@ Service catalog magento stack
...
+<img src="docs/diagram.jpeg" width="400">
+</p>
 
-* Having a VPC with private & public subnets containing a bastion server that can access instances by SSH
-* Having an S3 bucket for terraform remote states
-* Having an S3 bucket for magento code WITH versioning enable
+  * **ELB**: Amazon Elastic Load Balancing
+  * **EC2** Amazon Elastic Compute Cloud
+  * **RDS** Amazon Relational Database Service
+  * **Cache** Amazon Elasticache Redis
```
Comment (sdurrheimer) [inline README.md]:
```suggestion
  * **Cache** Amazon ElastiCache Redis
```

File: README.md [hunk 5]
```diff
@@ -4,53 +4,139 @@ Service catalog magento stack
...
-**full-deploy-front:**
-Ansible job that will fully install the EC2 instances, users, config, etc.
+In order to run this task, couple elements are required within the infrastructure:
 
-**app-deploy-front:**
-Ansible job meant to only deploy the Magento code in case of deployment.
+  * Having a VPC with private & public subnets [Here](https://docs.aws.amazon.com/vpc/latest/userguide/getting-started-ipv4.html#getting-started-create-vpc)
+  * Having a bastion server to run Ansible like described [Here](https://docs.cycloid.io/advanced-guide/ansible-integration.html#standard-usage)
+  * Having an S3 bucket to store Terraform remote states [Here](https://docs.aws.amazon.com/quickstarts/latest/s3backup/step-1-create-bucket.html)
+  * Having an S3 bucket for magento code **WITH** versioning enable [Here](https://docs.aws.amazon.com/quickstarts/latest/s3backup/step-1-create-bucket.html)
```
Comment (sdurrheimer) [inline README.md]:
```suggestion
  * Having an S3 bucket for Magento code **WITH** versioning enable [here](https://docs.aws.amazon.com/quickstarts/latest/s3backup/step-1-create-bucket.html)
```

File: README.md [hunk 6]
```diff
@@ -4,53 +4,139 @@ Service catalog magento stack
...
-**terraform-destroy:**
-Terraform job meant to destroy the whole stack - **NO CONFIRMATION ASKED**. If triggered, the full project **WILL** be destroyed.
-Use with caution.
+<p align="center">
+<img src="docs/pipeline.png" width="800">
+</p>
+
+**Jobs description**
+
+  * `build`: Runs the appropriate php/composer commands to build the magento code.
```
Comment (sdurrheimer) [inline README.md]:
```suggestion
  * `build`: Runs the appropriate `php/composer` commands to build the Magento code.
```

File: pipeline/variables.sample.yml [hunk 7]
```diff
@@ -3,51 +3,103 @@
...
+#. terraform_storage_bucket_path (required): ($ project $)/($ environment $)
+#+ AWS S3 bucket path to store terraform remote state file.
 terraform_storage_bucket_path: ($ project $)/($ environment $)
 
 #
 # Ansible
 #
 
+#. magento_deploy_bucket_name (required): ($ project $)-deploy
+#+ AWS S3 bucket name in which store the builded code of magento.
```
Comment (sdurrheimer) [inline pipeline/variables.sample.yml]:
```suggestion
#+ AWS S3 bucket name in which we will store your magento built code.
```

File: pipeline/variables.sample.yml [hunk 8]
```diff
@@ -3,51 +3,103 @@
...
 magento_deploy_bucket_name: ($ project $)-deploy
+
+#. magento_deploy_bucket_object_path (required): /catalog-magento/($ environment $)/magento.tar.gz
+#+ AWS S3 bucket path in which store the builded code of magento.
```
Comment (sdurrheimer) [inline pipeline/variables.sample.yml]:
```suggestion
#+ AWS S3 bucket path in which we will store your magento built code.
```

File: pipeline/variables.sample.yml [hunk 9]
```diff
@@ -3,51 +3,103 @@
...
 magento_deploy_bucket_object_path: /catalog-magento/($ environment $)/magento.tar.gz
 
-# Password used for your ansible vault variables (raw_ansible_vault_password depend of you credential name).
+#. ansible_vault_password (required): ((raw_ansible_vault_password))
+#+ Password used by ansible vault to decrypt your vaulted files.
```
Comment (sdurrheimer) [inline pipeline/variables.sample.yml]:
```suggestion
#+ Password used by `ansible-vault` to decrypt your vaulted files.
```

File: pipeline/variables.sample.yml [hunk 10]
```diff
@@ -3,51 +3,103 @@
...
 ansible_vault_password: ((raw_ansible_vault_password))
 
-# Configuration of the bastion server used to run ansible (ssh_bastion depend of you credential name)
+#. bastion_url (required): admin@bastion.cycloid.io
+#+ bastion url used by ansible to connect on aws ec2 instances.
```
Comment (sdurrheimer) [inline pipeline/variables.sample.yml]:
```suggestion
#+ bastion URL used by Ansible to connect on AWS EC2 instances.
```

File: pipeline/variables.sample.yml [hunk 11]
```diff
@@ -3,51 +3,103 @@
...
 bastion_url: admin@bastion.cycloid.io
+
+#. bastion_private_key_pair (required): ((ssh_bastion.ssh_key))
+#+ bastion SSH private key used by ansible to connect on aws ec2 instances and the bastion itself.
```
Comment (sdurrheimer) [inline pipeline/variables.sample.yml]:
```suggestion
#+ bastion SSH private key used by Ansible to connect on AWS EC2 instances and the bastion itself.
```

File: pipeline/variables.sample.yml [hunk 12]
```diff
@@ -3,51 +3,103 @@
...
 #
 # Repos
 #
 
+#. config_git_repository (required): git@github.com:MyUser/config-magento.git
+#+ Git repository url containing the config of the stack.
 config_git_repository: git@github.com:MyUser/config-magento.git
+
+#. config_git_branch (required): master
+#+ Branch of the config git repository.
```
Comment (sdurrheimer) [inline pipeline/variables.sample.yml]:
```suggestion
#+ Branch of the config Git repository.
```

File: pipeline/variables.sample.yml [hunk 13]
```diff
@@ -3,51 +3,103 @@
...
 config_git_branch: master
+
+#. config_git_private_key (required): ((git_config.ssh_key))
+#+ SSH key pair to fetch the config git repository.
```
Comment (sdurrheimer) [inline pipeline/variables.sample.yml]:
```suggestion
#+ SSH key pair to fetch the config Git repository.
```

File: pipeline/variables.sample.yml [hunk 14]
```diff
@@ -3,51 +3,103 @@
...
 config_git_private_key: ((git_config.ssh_key))
 
+#. magento_git_repository (required): git@github.com:MyUser/code-magento.git
+#+ Url to the git repository containing Magento source code.
```
Comment (sdurrheimer) [inline pipeline/variables.sample.yml]:
```suggestion
#+ URL to the Git repository containing Magento source code.
```

File: pipeline/variables.sample.yml [hunk 15]
```diff
@@ -3,51 +3,103 @@
...
 magento_git_repository: git@github.com:MyUser/code-magento.git
+
+#. magento_git_branch (required): master
+#+ Branch of the magento source code git repository.
```
Comment (sdurrheimer) [inline pipeline/variables.sample.yml]:
```suggestion
#+ Branch of the Magento source code Git repository.
```

File: pipeline/variables.sample.yml [hunk 16]
```diff
@@ -3,51 +3,103 @@
...
 magento_git_branch: master
+
+#. magento_git_private_key (required): ((git_magento.ssh_key))
+#+ SSH key pair to fetch magento source code git repository.
```
Comment (sdurrheimer) [inline pipeline/variables.sample.yml]:
```suggestion
#+ SSH key pair to fetch Magento source code Git repository.
```

File: pipeline/variables.sample.yml [hunk 17]
```diff
@@ -3,51 +3,103 @@
...
+#+ Name of the project.
 project: ($ project $)
+
+#. customer (required): ($ organization_canonical $)
+#+ Name of the Cycloid Organization, used as customer variable name.
 customer: ($ organization_canonical $)
 
-# If you need to use a specific branch of stack
+#. stack_git_branch (required): master
+#+ Branch to use on the public stack git repository
```
Comment (sdurrheimer) [inline pipeline/variables.sample.yml]:
```suggestion
#+ Branch to use on the public stack Git repository
```

File: pipeline/variables.sample.yml [hunk 18]
```diff
@@ -3,51 +3,103 @@
...
 stack_git_branch: master
 
-# Path of your configuration
+#. config_ansible_path (required): ($ project $)/ansible
+#+ Path of Ansible files in the config git repository
```
Comment (sdurrheimer) [inline pipeline/variables.sample.yml]:
```suggestion
#+ Path of Ansible files in the config Git repository
```

File: pipeline/variables.sample.yml [hunk 19]
```diff
@@ -3,51 +3,103 @@
...
 config_ansible_path: ($ project $)/ansible
+
+#. config_terraform_path (required): ($ project $)/terraform/($ environment $)
+#+ Path of Terraform files in the config git repository
```
Comment (sdurrheimer) [inline pipeline/variables.sample.yml]:
```suggestion
#+ Path of Terraform files in the config Git repository
```

File: terraform/magento.tf.sample [hunk 20]
```diff
@@ -3,46 +3,122 @@ variable "rds_password" {
...
+  #. vpc_id (required):
+  #+ Amazon VPC id on which create each components.
   vpc_id                   = "<vpc-id>"
+
+  #. private_subnets_ids (required, array):
+  #+ Amazon subnets IDs on which create each components.
   private_subnets_ids      = ["private-subnets"]
+
+  #. magento_ssl_cert (required):
+  #+ ARN of an Amazon certificate from Certificate Manager.
```
Comment (sdurrheimer) [inline terraform/magento.tf.sample]:
```suggestion
  #+ ARN of a certificate from the Amazon Certificate Manager.
```

File: terraform/magento.tf.sample [hunk 21]
```diff
@@ -3,46 +3,122 @@ variable "rds_password" {
...
   magento_ssl_cert         = "<ssl-cert-arn>"
+
+  #. bastion_sg_allow (optional):
+  #+ Amazon source security group ID which will be allowed to connect on Magento front port 22 (ssh).
```
Comment (sdurrheimer) [inline terraform/magento.tf.sample]:
```suggestion
  #+ Amazon source security group ID which will be allowed to connect on Magento front port 22 (SSH).
```

File: terraform/magento.tf.sample [hunk 22]
```diff
@@ -3,46 +3,122 @@ variable "rds_password" {
...
   public_subnets_ids       = ["<public-subnets>"]
 
+
+  #. rds_password (optional): ChangeMePls
+  #+ Password of the RDS database.
   rds_password             = "${var.rds_password}"
 
-  # SSH keypair name to use to deploy ec2 instances
+  #. keypair_name (optional): demo
+  #+ SSH keypair name to use to deploy ec2 instances.
```
Comment (sdurrheimer) [inline terraform/magento.tf.sample]:
```suggestion
  #+ SSH keypair name to use to deploy EC2 instances.
```

File: terraform/magento.tf.sample [hunk 23]
```diff
@@ -3,46 +3,122 @@ variable "rds_password" {
...
+  #+ RDS backup retention period in days.
   # rds_backup_retention = 7
+
+  #. rds_parameters (optional):
+  #+ RDS parameters to assign to the RDS database.
   # rds_parameters       = ""
 
+
+  #. cache_subnet (optional): Automatically generated from private_subnets_ids
+  #+ AWS elasticache subnet name.
```
Comment (sdurrheimer) [inline terraform/magento.tf.sample]:
```suggestion
  #+ AWS ElastiCache subnet name.
```

File: terraform/magento.tf.sample [hunk 24]
```diff
@@ -3,46 +3,122 @@ variable "rds_password" {
...
   # cache_subnet                     = "cache-subnet-id"
+
+  #. elasticache_type (optional): cache.t2.micro
+  #+ AWS elasticache instance type.
```
Comment (sdurrheimer) [inline terraform/magento.tf.sample]:
```suggestion
  #+ AWS ElastiCache instance type.
```

File: terraform/magento.tf.sample [hunk 25]
```diff
@@ -3,46 +3,122 @@ variable "rds_password" {
...
   # elasticache_type                 = "cache.t2.micro"
+
+  #. elasticache_nodes (optional): 1
+  #+ Number of AWS elasticache instances.
```
Comment (sdurrheimer) [inline terraform/magento.tf.sample]:
```suggestion
  #+ Number of AWS ElastiCache instances.
```

File: terraform/magento.tf.sample [hunk 26]
```diff
@@ -3,46 +3,122 @@ variable "rds_password" {
...
   # elasticache_nodes                = "1"
-  # elasticache_parameter_group_name = "default.redis4.0"
+
+  #. elasticache_parameter_group_name (optional): default.redis3.2
+  #+ AWS elasticache parameter group name.
```
Comment (sdurrheimer) [inline terraform/magento.tf.sample]:
```suggestion
  #+ AWS ElastiCache parameter group name.
```

File: terraform/magento.tf.sample [hunk 27]
```diff
@@ -3,46 +3,122 @@ variable "rds_password" {
...
+  # elasticache_parameter_group_name = "default.redis3.2"
+
+  #. elasticache_engine (optional): redis
+  #+ AWS elasticache engine.
```
Comment (sdurrheimer) [inline terraform/magento.tf.sample]:
```suggestion
  #+ AWS ElastiCache engine.
```

File: terraform/magento.tf.sample [hunk 28]
```diff
@@ -3,46 +3,122 @@ variable "rds_password" {
...
   # elasticache_engine               = "redis"
+
+  #. elasticache_engine (optional): 6379
+  #+ AWS elasticache binding port.
```
Comment (sdurrheimer) [inline terraform/magento.tf.sample]:
```suggestion
  #+ AWS ElastiCache binding port.
```

File: terraform/magento.tf.sample [hunk 29]
```diff
@@ -3,46 +3,122 @@ variable "rds_password" {
...
   # elasticache_port                 = "6379"
 
+  #. front_count (optional): 1
+  #+ Number of Aws EC2 frontend server to create.
```
Comment (sdurrheimer) [inline terraform/magento.tf.sample]:
```suggestion
  #+ Number of AWS EC2 frontend server to create.
```

File: terraform/magento.tf.sample [hunk 30]
```diff
@@ -3,46 +3,122 @@ variable "rds_password" {
...
   # front_count           = "1"
+
+  #. front_disk_size (optional): 60
+  #+ Disk size in Go of Aws EC2 frontend servers.
```
Comment (sdurrheimer) [inline terraform/magento.tf.sample]:
```suggestion
  #+ Disk size in Go of AWS EC2 frontend servers.
```

File: terraform/magento.tf.sample [hunk 31]
```diff
@@ -3,46 +3,122 @@ variable "rds_password" {
...
   # front_disk_size       = "60"
-  # front_disk_type       = "gp2"
+
+  #. front_type (optional): t3.small
+  #+ Type of Aws EC2 frontend servers.
```
Comment (sdurrheimer) [inline terraform/magento.tf.sample]:
```suggestion
  #+ Type of AWS EC2 frontend servers.
```

File: terraform/magento.tf.sample [hunk 32]
```diff
@@ -3,46 +3,122 @@ variable "rds_password" {
...
   # front_type            = "t3.small"
+
+  #. front_ebs_optimized (optional, bool): false
+  #+ Whether the Instance is EBS optimized or not, related to the instance type you choose.
```
Comment (sdurrheimer) [inline terraform/magento.tf.sample]:
```suggestion
  #+ Whether the instance is EBS optimized or not, related to the instance type you choose.
```

File: terraform/magento.tf.sample [hunk 33]
```diff
@@ -3,46 +3,122 @@ variable "rds_password" {
...
   # rds_multiaz          = false
-  # rds_storage_type     = "gp2"
-  # rds_subnet           = "<rds-subnet-id>"
+
+  #. rds_subnet (optional): Automatically generated from private_subnets_ids
+  #+ ID of the private DB subnet group to use for RDS database.
+  # rds_subnet           = "<rds-subnet-group>"
+
+  #. rds_type (optional): db.t2.small
+  #+ AWS Instance type of the RDS database.
```
Comment (sdurrheimer) [inline terraform/magento.tf.sample]:
```suggestion
  #+ AWS instance type of the RDS database.
```

File: terraform/outputs.tf [hunk 34]
```diff
@@ -1,31 +1,39 @@
 output "elb_front_dns_name" {
-  value = "${module.magento.elb_front_dns_name}"
+  value       = "${module.magento.elb_front_dns_name}"
+  description = "DNS name of the front elb."
```
Comment (sdurrheimer) [inline terraform/outputs.tf]:
```suggestion
  description = "DNS name of the front ELB."
```

File: terraform/outputs.tf [hunk 35]
```diff
@@ -1,31 +1,39 @@
...
 }
 
 output "elb_front_zone_id" {
-  value = "${module.magento.elb_front_zone_id}"
+  value       = "${module.magento.elb_front_zone_id}"
+  description = "Zone ID of the front elb."
```
Comment (sdurrheimer) [inline terraform/outputs.tf]:
```suggestion
  description = "Zone ID of the front ELB."
```

File: terraform/outputs.tf [hunk 36]
```diff
@@ -1,31 +1,39 @@
...
 output "front_private_ips" {
-  value = "${module.magento.front_private_ips}"
+  value       = "${module.magento.front_private_ips}"
+  description = "Private IPs of the front EC2 server."
 }
 
 output "cache_address" {
-  value = "${module.magento.cache_address}"
+  value       = "${module.magento.cache_address}"
+  description = "Address of the elasticache."
```
Comment (sdurrheimer) [inline terraform/outputs.tf]:
```suggestion
  description = "Address of the ElastiCache."
```

File: README.md [hunk 37]
```diff
@@ -4,53 +4,139 @@ Service catalog magento stack
...
-Dummy job meant to eventually be replaced by proper tests or removed.
+# Requirements
 
-**full-deploy-front:**
-Ansible job that will fully install the EC2 instances, users, config, etc.
+In order to run this task, couple elements are required within the infrastructure:
 
-**app-deploy-front:**
-Ansible job meant to only deploy the Magento code in case of deployment.
+  * Having a VPC with private & public subnets [Here](https://docs.aws.amazon.com/vpc/latest/userguide/getting-started-ipv4.html#getting-started-create-vpc)
```
Comment (sdurrheimer) [inline README.md]:
```suggestion
  * Having a VPC with private & public subnets [here](https://docs.aws.amazon.com/vpc/latest/userguide/getting-started-ipv4.html#getting-started-create-vpc)
```

File: README.md [hunk 38]
```diff
@@ -4,53 +4,139 @@ Service catalog magento stack
...
+  * Having a bastion server to run Ansible like described [Here](https://docs.cycloid.io/advanced-guide/ansible-integration.html#standard-usage)
```
Comment (sdurrheimer) [inline README.md]:
```suggestion
  * Having a bastion server to run Ansible like described [here](https://docs.cycloid.io/advanced-guide/ansible-integration.html#standard-usage)
```

File: README.md [hunk 39]
```diff
@@ -4,53 +4,139 @@ Service catalog magento stack
...
+  * Having an S3 bucket to store Terraform remote states [Here](https://docs.aws.amazon.com/quickstarts/latest/s3backup/step-1-create-bucket.html)
```
Comment (sdurrheimer) [inline README.md]:
```suggestion
  * Having an S3 bucket to store Terraform remote states [here](https://docs.aws.amazon.com/quickstarts/latest/s3backup/step-1-create-bucket.html)
```

File: README.md [hunk 40]
```diff
@@ -4,53 +4,139 @@ Service catalog magento stack
...
-Same as unittest, that's a dummy job to eventually ensure everything is working as expected following the deployment.
+# Details
 
-**terraform-plan:**
-Terraform job that will simply make a plan of the infrastructure's stack.
+## Pipeline
 
-**terraform-apply:**
-Terraform job similar to the plan one, but will actually create/update everything that needs to. Please see the plan diff for a better understanding.
+> **Note** The pipeline contains a manual approval between terraform plan and terraform apply.
```
Comment (sdurrheimer) [inline README.md]:
```suggestion
> **Note** The pipeline contains a manual approval between `terraform plan` and `terraform apply`.
```

File: README.md [hunk 41]
```diff
@@ -4,53 +4,139 @@ Service catalog magento stack
...
+> That means if you trigger a terraform plan, to apply it, you have to go on terraform apply job
```
Comment (sdurrheimer) [inline README.md]:
```suggestion
> That means if you trigger a `terraform plan`, to apply it, you have to go on `terraform apply` job
```

File: pipeline/variables.sample.yml [hunk 42]
```diff
@@ -3,51 +3,103 @@
 #
 
 # Amazon access to use to run terraform. Usually admin account to be able to create Amazon services.
+
+#. aws_access_key (required): ((aws.access_key))
+#+ Amazon AWS access key for Terraform. see value format [Here](https://docs.cycloid.io/advanced-guide/integrate-and-use-cycloid-credentials-manager.html#vault-in-the-pipeline)
```
Comment (sdurrheimer) [inline pipeline/variables.sample.yml]:
```suggestion
#+ Amazon AWS access key for Terraform. see value format [here](https://docs.cycloid.io/advanced-guide/integrate-and-use-cycloid-credentials-manager.html#vault-in-the-pipeline).
```

File: pipeline/variables.sample.yml [hunk 43]
```diff
@@ -3,51 +3,103 @@
...
 aws_access_key: ((aws.access_key))
+
+#. aws_secret_key (required): ((aws.secret_key))
+#+ Amazon AWS secret key for Terraform. see value format [Here](https://docs.cycloid.io/advanced-guide/integrate-and-use-cycloid-credentials-manager.html#vault-in-the-pipeline)
```
Comment (sdurrheimer) [inline pipeline/variables.sample.yml]:
```suggestion
#+ Amazon AWS secret key for Terraform. see value format [here](https://docs.cycloid.io/advanced-guide/integrate-and-use-cycloid-credentials-manager.html#vault-in-the-pipeline).
```

File: ansible/environments/front.yml.sample [hunk 44]
```diff
@@ -1,13 +1,26 @@
...
-# magento_admin_email: root@localhost.local
-# magento_admin_user: admin
-
-# magento_admin_password: 4Bigs3cr3t
+#
+# Magento config
+#
+
+#. magento_backend_frontname: admin
+#+ Magento frontName to use in env.php
```
Comment (sdurrheimer) [inline ansible/environments/front.yml.sample]:
```suggestion
#+ Magento frontName to use in env.php.
```

File: ansible/environments/front.yml.sample [hunk 45]
```diff
@@ -1,13 +1,26 @@
...
+
+#. magento_admin_firstname: admin
+#+ Magento install option admin-firstname
```
Comment (sdurrheimer) [inline ansible/environments/front.yml.sample]:
```suggestion
#+ Magento install option admin-firstname.
```

File: ansible/environments/front.yml.sample [hunk 46]
```diff
@@ -1,13 +1,26 @@
...
+
+#. magento_admin_lastname: admin
+#+ Magento install option admin-lastname
```
Comment (sdurrheimer) [inline ansible/environments/front.yml.sample]:
```suggestion
#+ Magento install option admin-lastname.
```

File: ansible/environments/front.yml.sample [hunk 47]
```diff
@@ -1,13 +1,26 @@
...
+
+#. magento_admin_email: root@localhost.local
+#+ Magento install option admin-email
```
Comment (sdurrheimer) [inline ansible/environments/front.yml.sample]:
```suggestion
#+ Magento install option admin-email.
```

File: ansible/environments/front.yml.sample [hunk 48]
```diff
@@ -1,13 +1,26 @@
...
+
+#. magento_admin_user: admin
+#+ Magento install option admin-user
```
Comment (sdurrheimer) [inline ansible/environments/front.yml.sample]:
```suggestion
#+ Magento install option admin-user.
```

File: ansible/environments/front.yml.sample [hunk 49]
```diff
@@ -1,13 +1,26 @@
...
+
+#. magento_admin_password: 4Bigs3cr3t
+#+ Magento install option admin-password
```
Comment (sdurrheimer) [inline ansible/environments/front.yml.sample]:
```suggestion
#+ Magento install option admin-password.
```

**Pull request merged**